src/Model/MyParcelRequest.php

```diff
@@ -167,7 +168,13 @@ public function sendRequest($method = 'POST', $uri = self::REQUEST_TYPE_SHIPMENT
         $request->close();
 
         if ($this->getError()) {
-            throw new ApiException('Error in MyParcel API request: ' . $this->getError() . ' Url: ' . $url . ' Request: ' . $this->body);
+            switch ($this->errorCode->first()) {
+                case 3716:
+                    throw new ApiException('Error ' . $this->errorCode->first() . ' Your account needs to be activated by MyParcel!');
```

Comment:
Je moet even deze exception zijn eigen class geven. 
```suggestion
                    throw new AccountNotActiveException('Error ' . $this->errorCode->first() . ' Your account needs to be activated by MyParcel!');
```

Reply:
Ik zou geen ! gebruiken